### ISSUE TITLE
Add Device Orientation Demo to Slides

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#195]
```
The new Device Orientation APIs are a great crowd pleaser and really show the 
deep integration with the host device.
```

Original issue reported on code.google.com by `sethladd@google.com` on 16 Sep 2010 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
I have cool demo for this.
```

Original comment by `ericbide...@html5rocks.com` on 17 Sep 2010 at 2:01
- Added labels: **Milestone-X**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 0d1779b69e.
```

Original comment by `ericbide...@html5rocks.com` on 10 Nov 2010 at 1:46
- Changed state: **Fixed**
